README.md

```diff
@@ -350,6 +351,31 @@ keymap:
 
 Note how Alt-f and Alt-b work in all apps, but the definition of Alt-f is slightly different in LibreOffice Writer. When that app is active, the first definition overrides the second definition; but for any other app, only the second definition is found. This is because xremap uses the first matching definition that it finds.
 
+### device
```

Comment:
:+1: Nice documentation. For better discoverability, similar to `application`, could you mention this under `modmap` and `keymap` as well? (like the `application: # Optional` part)

Reply:
Will do!

Reply:
Not sure I'll get to this today though, probably tomorrow. Feel free to merge today or wait, either is fine by me

Reply:
No worries, I'll just release the current version first :+1: